float64[7] dq_arm

